build_openroad.sh

```diff
@@ -137,6 +138,12 @@ elif [ "$build_method" == "LOCAL" ]; then
 
   mkdir -p tools/build/OpenROAD
   (cd tools/build/OpenROAD && cmake ../../OpenROAD && $NICE make -j$PROC)
+
+  mkdir -p tools/build/LSOracle
+  cmake -B tools/build/LSOracle tools/LSOracle -D CMAKE_BUILD_TYPE=RELEASE -D YOSYS_INCLUDE_DIR=$(pwd)/tools/yosys -D YOSYS_PLUGIN=ON
```

Comment:
I'm concerned that pwd could be anywhere; they don't have to run this script from this directory.

Reply:
As I look at the rest of the script seems to assume this so never mind.

Reply:
If it was run from elsewhere, every other relative path in this script would break, so it does need to be run from the root directory.

From directory above:
```./OpenROAD-flow-scripts/build_openroad.sh 
[INFO][FLOW-0000] using docker build method. This will create a docker image tagged 'openroad/flow-scripts'
fatal: not a git repository (or any of the parent directories): .git```

From tools:
```
../build_openroad.sh 
[INFO][FLOW-0000] using docker build method. This will create a docker image tagged 'openroad/flow-scripts'
unable to prepare context: path "tools/yosys" not found
```
This path could be relative, but it would need to be relative to the CMakeLists.txt in tools/LSOracle/yosys-plugin (unless there's some way in cmake to change that behavior) It seemed more clear to pass an absolute path rather than a fairly random looking relative path (../../../yosys)